tests/Export/Adapters/ExportItemAdapterTest.php

```diff
@@ -26,6 +26,11 @@ class ExportItemAdapterTest extends TestCase
     use ProductHelper;
     use ServicesHelper;
 
+    public function setUp(): void
+    {
+        $this->pluginConfig = $this->getPluginConfig();
```

Comment:
You can remove this here and within the test, as you are only using the default one anyway.